src/marqo/core/models/marqo_index.py

```diff
@@ -476,7 +477,7 @@ def validate_structured_field(values, marqo_index: bool) -> None:
             f'{FieldType.MultimodalCombination.value}'
         )
 
-    if FieldFeature.LexicalSearch in features and type not in [FieldType.Text, FieldType.ArrayText]:
+    if FieldFeature.LexicalSearch in features and type not in [FieldType.Text, FieldType.ArrayText, FieldType.CustomVector]:
```

Comment:
Yes, there may be cases where a user wants to lexically search for the content associated with the custom vector. For example:

```
{
    "content": "podcast audio from Wednesday",
    "vector": [0.1, 0.2.....]
}
```

It could be useful to lexical search "podcast audio"